projects/kavaBoost.js

```diff
@@ -0,0 +1,35 @@
+const utils = require('./helper/utils');
+const sdk = require('@defillama/sdk')
+
+async function tvl(timestamp) {
+  let balances = {};
+  let url = `https://api2.kava.io/kava/savings/v1beta1/total_supply`;
+
+  if (Math.abs(Date.now()/1000 - timestamp) > 3600){
+    const block = await sdk.api.util.lookupBlock(timestamp, {chain:'kava'})
+    url += `?height=${block.block}`
+  }
+
+  const response = await utils.fetchURL(url);
+
+  for (let coin of response.data.result) {
+    const tokenInfo = generic(coin.denom);
+    const tokenName = coin.denom === 'bkava' ? 'bkava' : tokenInfo[0];
+    balances[tokenName] = coin.amount / 10 ** tokenInfo[1];
```

Comment:
bKAVA is being added to `balances` within the `for` loop

```
{ 
  'bkava: 25586386.019095, 
  'usd-coin': 1187281.342515, 
  'kava': 25000000 
}
```

but it isn't returned when running `node test projects/kavaBoost.js`

```
--- kava ---
KAVA                      28.00 M
USDC                      1.19 M
Total: 29.19 M 

--- tvl ---
KAVA                      28.00 M
USDC                      1.19 M
Total: 29.19 M 

------ TVL ------
kava                      29.19 M

total                    29.19 M 
```